app/components/ui/PopUpMessage.scss

```diff
@@ -0,0 +1,41 @@
+.pop-up-message {
+  display: table;
+  position: fixed;
+  margin: auto;
+  top: 100px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  color: white;
+  height: 50px;
+  width: 300px;
+  text-align: center;
+  border-radius: 4px;
+  padding-top: 4px;
+  padding-bottom: 4px;
+  transition: opacity 0.5s;
+  -webkit-transition: opacity 0.5s; /* Safari */
+
+  &.success {
+    background-color: #5BBA81;
```

Comment:
We have some css variables defined in `_helpers.scss`. If possible, it would be great to add these colors as variables in there and then we can just reference the variable here. I noticed that we also use these colors in `_layout.scss` so we could replace the hardcoded hex values with these new variables as well. That way, if we ever want to change the colors, they are only defined in one place.

Reply:
Great idea! I'll go ahead and do that.